README.md

```diff
@@ -1,41 +1,39 @@
 # Coding Challenge
-Unsere Redakteure müssen Tabellen mit Produktvergleichen verwalten (z.B. https://widget.vgl.org/?comparisonId=2907). 
-Dazu müssen sie unter anderem in der Lage sein, einzelne Produkte anlegen, bearbeiten und löschen zu können. 
-Immer wenn ein Produkt angelegt, bearbeitet oder gelöscht worden ist, müssen verschiedene Prozesse angestoßen und durchlaufen werden, 
-z.B. müssen Vergleichstabellen mit den neuen Daten aktualisiert werden. 
-Da diese Prozesse sehr umfangreich werden können, dürfen diese nicht direkt an den redaktionellen Vorgang gebunden sein, 
-sondern müssen unabhängig davon passieren.
+Our editors have to manage tables which compare products or services (e.g. https://widget.vgl.org/?comparisonId=2907).
+To manage these tables, the editors have to be able to create, update and delete products used in the tables.
+Everytime a product is created, updated or deleted, different process have to be executed. For example if a product is updated, all tables using this product have to be updated.
+Because these processes can take a long time and should not block the work of the editors, they must not be connected directly to the editors workflows but run independently.
 
-Du sollst eine Software entwickeln, die die oben beschriebenen Geschäftsprozesse in einer vereinfachten Version wiederspiegeln.
+You must develop a software, which can handle this scenario.
```

Comment:
You're asked to develop a software, which can handle this scenario in a simplified manner.